packages/graphql/schemas/cloud.graphql

```diff
@@ -1,6 +1,72 @@
 ### This file was generated by Nexus Schema
 ### Do not make changes to this file directly
 
+"""
```

Comment:
Whats with the graph ql schema changes?

Reply:
someone made schema changes and didn't run `yarn`

Reply:
going to remove these as checking them in causes typing issues

Reply:
fixed in a8808634a3